test/e2e/appmesh.bats

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bats
+
+# App Mesh charts e2e test
+
+set -o errexit
+
+export KUBECONFIG="$(kind get kubeconfig-path --name="kind")"
+export REPO_ROOT=$(git rev-parse --show-toplevel)
+
+namespace=appmesh-system
+charts=${REPO_ROOT}/stable
+
+load ${REPO_ROOT}/test/lib/helm.sh
+
+function setup() {
+  infof "Preparing namespace $namespace"
+  kubectl create ns $namespace >&3
```

Comment:
Why are we using fd 3 here?

Reply:
This is how you can write to the bats test log.